inspirehep/utils/references.py

```diff
@@ -42,24 +44,21 @@ def references(self):
             refs_to_get_from_es = [
                 ref['recid'] for ref in references if ref.get('recid')
```

Comment:
I think it would work. The only concern was, how many requests to ES are done when calling `replace_refs` if there are 100 references? In this case there would be only one request doing the [search](https://github.com/inspirehep/inspire-next/pull/1141/files/83881fb1b490bd3b4dafa6a7afdc4007584e579e#diff-bc37d9f037fdf5e12a93242bc0ee6b91R49)

Also I think I am going to modify this code and call directly `es.search()` so that I can pass the `_source` parameter and only get back the fields I really need (like `title` and `authors`)